.ci/docker/ci_commit_pins/triton.txt

```diff
@@ -1 +1 @@
-a9bc1a36470eefafe0e2ab2503b8698f1e89e7e3
+release/2.2.x
```

Comment:
@shunting314 this will not work. Let me handle enabling 2.2 in CI. I would need to revert https://github.com/pytorch/pytorch/pull/121765 and builder PR first.